packages/finder/src/reporters/xcode.ts

```diff
@@ -0,0 +1,26 @@
+import {IClone, IOptions} from '@jscpd/core';
+import {IReporter} from '..';
+import {getPath} from '../utils/reports';
+
+export class XcodeReporter implements IReporter {
+	constructor(private readonly options: IOptions) {
+	}
+
+	public report(clones: IClone[]): void {
+		clones.forEach((clone: IClone) => {
+			this.cloneFound(clone);
+		});
+		console.log(`Found ${clones.length} clones.`);
+	}
+
+	private cloneFound(clone: IClone): void {
+		const pathA = getPath(clone.duplicationA.sourceId, this.options);
```

Comment:
if we are going to use absolute path here we can use following code:
```typescript
const pathA = getPath(clone.duplicationA.sourceId, {...this.options, absolute:true});
```

Reply:
I noticed that this only works if the full path of the files to analyze is passed from the command line. If only the relative path is passed, this function returns the relative path.